File: C++ Programming/problems on file handling/problem10.cpp
/*
10. Write a program which accepts file name and one count from user and read that number 
	  of characters from starting position. 
	
	Input : Demo.txt  12 
	
	Output : Display first 12 characters from Demo.txt 
	
	Author : Milind Nalawade

	Date : 1/09/2021

	Language : C++ programming

*/
using namespace std;
#include<iostream>
#include<stdlib.h>
#include<unistd.h>
#include<fcntl.h>

//Entry point function
int main()
{
	int fd=0,ret=0,cnt=0,i=0,no=0;
	char arr[30]={'\0'};
	char brr[100]={'\0'};
	cout<<"\n Enter the file name : ";
	cin.getline(arr,30);
	cout<<"\n Enter your number : ";
	cin>>no;
	fd=open(arr,O_RDONLY,0777);
	ret=read(fd,brr,no);
	cout<<brr;
	return 0;

}

File: Python Programming/problems on file handling/problem10.py
'''
10. Write a program which accepts file name and one count from user and read that number 
	  of characters from starting position. 
	
	Input : Demo.txt  12 
	
	Output : Display first 12 characters from Demo.txt 

	Author : Milind Nalawade

	Date : 01/09/2021

	Language : Python 3.9.0
	
'''
import os
#=============
#Entry point function
#=============
def main():
	str=input("\n Enter file name : ")
	fd=open(str,"r")
	no=int(input("\n Enter your number : "))
	if fd:
		print("-"*50)
		print(" File opened successfully !!!")
		print("-"*50)
	else:
		print(" File not found !!!")
	line=fd.read(no)
	
	print("\n Output : ",line)	

#=====
#starter
#=====
if __name__=="__main__":
	main()
	